Selenium/SeleniumTests.cs

```diff
@@ -1,17 +1,102 @@
-﻿using NUnit.Framework;
+﻿using Microsoft.VisualStudio.TestPlatform.CommunicationUtilities;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+using OpenQA.Selenium;
 using OpenQA.Selenium.Chrome;
+using System.Text;
+using OpenQA.Selenium.Support.UI;
+using System.ComponentModel;
+using System.Linq;
 
 namespace Selenium
 {
+    [TestFixture]
     public class SeleniumTests
     {
-        [Test]
-        public void TestWebsiteTitle()
+        private IWebDriver driver;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            driver = new ChromeDriver();
+        }
+        [OneTimeTearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+        [Test, Order(1)]
+        public void SuccessfullLoginAndItemOpening()
         {
-            var driver = new ChromeDriver();
-            driver.Navigate().GoToUrl("")
-            driver.Navigate().GoToUrl("https://www.example.com");
+            driver.Navigate().GoToUrl("https://www.saucedemo.com/");
+            var elementUserName = driver.FindElement(By.Id("user-name"));
+            elementUserName.SendKeys("standard_user");
+            var elementPassword = driver.FindElement(By.Id("password"));
+            elementPassword.SendKeys("secret_sauce");
+
+            var elementLoginButton = driver.FindElement(By.Id("login-button"));
+            elementLoginButton.Click();
+
+            string pageUrl = driver.Url;
+            Assert.That(pageUrl, Is.EqualTo("https://www.saucedemo.com/inventory.html"));
+            string pageTitle = driver.Title;
+            Assert.That(pageTitle, Is.EqualTo("Swag Labs"));
+
+            var elementItem = driver.FindElement(By.Id("item_4_title_link"));
+            elementItem.Click();
+
+            string itemUrl = driver.Url;
+            Assert.That(itemUrl, Is.EqualTo("https://www.saucedemo.com/inventory-item.html?id=4"));
+
+            var itemTitle = driver.FindElement(By.CssSelector("[class*='inventory_details_name']"));
+            Assert.That(itemTitle.Text.Contains("Sauce Labs Backpack"));
 
+            var itemDescription = driver.FindElement(By.CssSelector("[class*='inventory_details_desc']"));
+            Assert.That(itemDescription.Text.Contains("streamlined Sly Pack"));
+
+            var itemPrice = driver.FindElement(By.ClassName("inventory_details_price"));
+            Assert.That(itemDescription.Text.Contains("$29.99"));
+        }
+        [Test, Order(2)]
+        public void FailedLoginWithEmptyCredentials()
+        {
+            driver.Navigate().GoToUrl("https://www.saucedemo.com/");
+            var elementUserName = driver.FindElement(By.Id("user-name"));
+            var elementPassword = driver.FindElement(By.Id("password"));
```

Comment:
Could you please add check for displaying elements or check that input fields are empty

Reply:
fixed